bedrock/legal/templates/legal/fraud-report.html

```diff
@@ -27,6 +27,20 @@ <h3 class="mzp-c-sidemenu-label">{{ self.page_title() }}</h3>
 {% endblock %}
 
 {% block article %}
+
+{% if form_submitted %}
+<aside class="mzp-c-notification-bar mzp-t-success">
+  <p><strong>{{ _('Your report has been sent.') }}</strong></p>
+  <p>{{ _('Thank you for your help. It’s our community that makes us great.') }}</p>
+</aside>
+{% endif %}
+
+{% if form_error %}
```

Comment:
Same applies for the error notification:

![image](https://user-images.githubusercontent.com/400117/89388949-a5346b00-d6fc-11ea-8efe-e9f394c8e1db.png)